src/team/memberView.js

```diff
@@ -1,20 +1,19 @@
 import React from 'react'
-import teamMembersObj from './teamMembers'
-// A React element that displays all team members
+import {teamMembersObj} from './teamMembers'
 
-export default (member)  => 
-  (
-    <div className="col-md-4 col-xs-6 work-item web-design mockups">
-      <div className="work-container">
-       <a href={"/team/" + member}>
-        <div className="work-img">
-          <img src={teamMembersObj[member].memberImg} alt=""/>
-        </div>
-        </a>
-        <div className="work-description">
-          <h3><a href={"/team/" + member}>{teamMembersObj[member].memberFullName}</a></h3>
-          <span><a href={"/team/" + member}>{teamMembersObj[member].memberRole}</a></span>
-        </div>
+// A React element that displays a team member
+
+export default (member, styles)  => 
+  <div className={styles}>
```

Comment:
IIUC, this is the element we should try to move into the page.  If we can't then we should change the selectors that target this element.